cicd/pipelines/running-workloads-and-buildah-as-user-namespaces-on-openshift-pipelines.adoc

```diff
@@ -0,0 +1,20 @@
+:_content-type: ASSEMBLY
+[id="running-workloads-and-buildah-as-user-namespaces-on-openshift-pipelines"]
+= Running workloads and Buildah as user namespaces on {pipelines-title}
+include::_attributes/common-attributes.adoc[]
+:context: running-workloads-and-buildah-as-user-namespaces-on-openshift-pipelines
+
+toc::[]
+
+Running {pipelines-shortname} as root user on a container can expose the host and container processes to other resources. You can reduce this type of exposure by using one of the following methods:
```

Comment:
```suggestion
Running {pipelines-shortname} as the root user on a container can expose the host and container processes to other resources. You can reduce this type of exposure by using one of the following methods:
```
- needs a definite article
- Would it be useful to mention that the `other resources` could be potentially malicious? It suppose it's implied with the current phrasing, but I think it wouldn't hurt to make it explicit. WDYT?